.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
+
 matrix:
   fast_finish: true
 
-# Only build pushes to the master branch (and PRs)
```

Comment:
Pulling this filter causes duplicate builds of every PR, and duplicate comments.